src/logserver/server.py

```diff
@@ -133,6 +147,8 @@ def get_next_logline(self) -> str | None:
             return self.data_queue.get()
         return None
 
+    # TODO: Add a close method
```

Comment:
Is this still open?

Reply:
It's not crucial, but yes, it's still open.

Reply:
We could handle this in a later pull request if desired

Reply:
Yes, we should. Closing the `LogServer` might still lead to some unexpected behavior. #24